packages/core/admin/admin/src/pages/MarketplacePage/tests/index.test.js

```diff
@@ -284,10 +284,10 @@ describe('Marketplace page', () => {
     expect(sortButton).toBeVisible();
   });
 
-  it('shows the correct options on sort select', () => {
+  it('shows the correct options on sort select', async () => {
```

Comment:
Why do you need to make it async? It doesn't look like we await anything

Reply:
woops I was trying a `waitFor` back with .click I'll remove it